lib/ansible/modules/cloud/scaleway/scaleway_lb.py

```diff
@@ -320,7 +320,7 @@ def core(module):
     }
     module.params['api_url'] = SCALEWAY_ENDPOINT
     api = Scaleway(module=module)
-    api.api_path = "lbaas/v1beta1/regions/%s/lbs" % region
+    api.api_path = "lb/v1/regions/%s/lbs" % region
```

Comment:
Can you also set the api version in a variable and set it by default to `v1` This let user be able to change the version without having to release again the module :)